front-end/src/screens/CreatePost/index.tsx

```diff
@@ -108,6 +111,12 @@ const CreatePost = ({ className }:Props): JSX.Element => {
 						rules={{ required: true }}
 					/>
 
+					<Form.Group className="hidden">
```

Comment:
Commenting is much better than fiddling with css, having to change the class, remove it in the style is more error prone.

Reply:
got it